charts/fint-adapter-sikri/templates/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.nameOverride }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Values.nameOverride }}
+  template:
+    metadata:
+    spec:
+      volumes:
+        - name: tempdir
+          emptyDir:
+            sizeLimit: 1G
+      containers:
+        - name: {{ .Values.nameOverride }}
+          image: "{{ .Values.image.repository }}"
+          #imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 8091
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /
+              port: http
+          readinessProbe:
+            httpGet:
+              path: /
+              port: http
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
```

Comment:
Do something like this:

```
         resources:
            limits:
              memory: {{ .Values.consumer.deployment.resources.limitsMemory }}
              cpu: {{ .Values.consumer.deployment.resources.limitsCpu }}
            requests:
              memory: {{ .Values.consumer.deployment.resources.requestsMemory }}
              cpu: {{ .Values.consumer.deployment.resources.requestsCpu }}
```

Ref. https://helm.sh/docs/chart_best_practices/values/#flat-or-nested-values